brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/accountverification/AccountVerificationViewModel.kt

```diff
@@ -144,4 +162,20 @@ class AccountVerificationViewModel(
             )
         }
     }
+
+    fun updateProfile() {
+        viewModelScope.launch(Dispatchers.IO) {
+            profileManager.updateProfile().collect { profile ->
+                if (profile != null) {
```

Comment:
It would be better to use early return like this:
`if(profile == null) return@collect`
It reduces nesting and improves code readability